nuxeo-liveconnect-dropbox-core/src/main/java/org/nuxeo/ecm/liveconnect/dropbox/DropboxBlobProvider.java

```diff
@@ -75,18 +79,17 @@ public URI getURI(ManagedBlob blob, UsageHint usage, HttpServletRequest servletR
         LiveConnectFileInfo fileInfo = toFileInfo(blob);
         String filePath = fileInfo.getFileId();
         String url = null;
-        DbxClient client = getDropboxClient(getCredential(fileInfo));
+        DbxClientV2 client = getDropboxClient(getCredential(fileInfo));
         try {
             switch (usage) {
```

Comment:
Question: when are all these cases called (DOWNLOAD, VIEW, STREAM, ... ) ? 
only able to reproduce "DOWNLOAD" and "EMBED" (Which is not even handled).